salt/orch/update-etc-hosts.sls

```diff
@@ -1,19 +1,19 @@
-{% if salt.saltutil.runner('mine.get', tgt='P@roles:kube-(master|minion) and G@bootstrap_complete:true', fun='caasp_fqdn', tgt_type='compound')|length > 0 %}
+{% if salt.saltutil.runner('mine.get', tgt='P@roles:kube-(master|minion) and G@bootstrap_complete:true and not G@update_in_progress:true', fun='caasp_fqdn', tgt_type='compound')|length > 0 %}
```

Comment:
A suggestion:

```jinja
{%- set updates_all_target = "P@roles:kube-(master|minion) and G@bootstrap_complete:true and not G@update_in_progress:true" %}
```

And use it everywhere like `- tgt: {{ updates_all_target }}`